src/events.ts

```diff
@@ -41,6 +41,11 @@ class EventRPCServerCreated extends EventRPCServer {}
 class EventRPCServerDestroy extends EventRPCServer {}
 
 class EventRPCServerDestroyed extends EventRPCServer {}
+class EventRPCServerStart extends EventRPCServer {}
+class EventRPCServerStarted extends EventRPCServer {}
+class EventRPCServerStopping extends EventRPCServer {}
+
+class EventRPCServerStopped extends EventRPCServer {}
```

Comment:
use proper spacing, there is an empty line between each definition by convention.